pkg/scripts_test/check.go

```diff
@@ -56,7 +56,13 @@ func checkRun(c check) {
 }
 
 func checkConfigDirectoryOwnershipAndPermissions(c check) {
-	PathHasOwner(c.test, etcPath, rootUser, rootGroup)
+	ownerUser := systemUser
+	ownerGroup := systemGroup
+	if !c.installOptions.useNativePackaging {
+		ownerUser = rootUser
+		ownerGroup = rootGroup
```

Comment:
Why is this different? Do the native packages not set the same owner as the install script?

Reply:
When `SYSTEMD_DISABLED=true`, the install script doesn't add the otelcol-sumo user and thus the files are owned by root. The packages only support systemd and thus will always make use of the otelcol-sumo user.